generator-theia/src/extension/extension-package-generator.ts

```diff
@@ -75,7 +84,7 @@ export class ExtensionPackageGenerator extends AbstractGenerator {
             ],
             "scripts": {
                 "clean": "yarn run compile:clean && yarn run docs:clean",
-                "build": "concurrently -n compile,lint -c blue,green \"yarn run compile\" \"yarn run lint\"",
+                "build": buildCmd,
```

Comment:
I think it would be better to have a function here and return the proper value rather than using a local var

Reply:
Done, created a method "getBuildCommand()"